examples/variational_autoencoder_deconv.py

```diff
@@ -15,21 +15,30 @@
 # input image dimensions
 img_rows, img_cols, img_chns = 28, 28, 1
 # number of convolutional filters to use
-nb_filters = 32
+nb_filters = 64
 # convolution kernel size
 nb_conv = 3
 
-batch_size = 16
+batch_size = 128
 original_dim = (img_chns, img_rows, img_cols)
 latent_dim = 2
 intermediate_dim = 128
-epsilon_std = 0.01
+epsilon_std = 1.0
```

Comment:
To be able to compare the results with the other variationnal autoencoder example I also changed the std.